changelogs/fragments/1427-backup_tag-add-new-module.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
I believe this can be `trivial` because a new module doesn't need a changelog fragment. It will get a separate entry in the changelog under new modules.